modules/images/webp-uploads/load.php

```diff
@@ -684,6 +684,135 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 }
 add_filter( 'rest_prepare_attachment', 'webp_uploads_update_rest_attachment', 10, 3 );
 
+/**
+ * Adds sources to metadata for an attachment.
+ *
+ * @since n.e.x.t
+ *
+ * @param array  $metadata              Metadata of the attachment.
+ * @param array  $valid_mime_transforms List of valid mime transforms for current image mime type.
+ * @param string $file                  Path to original file.
+ * @param array  $main_images           Path of all main image files of all mime types.
+ * @param array  $subsized_images       Path of all subsized image file of all mime types.
+ *
+ * @return array                        Metadata with sources added.
```

Comment:
There doesn't need to be a blank line here and the description doesn't need to be indented similar to the `@param` docs.
```suggestion
 * @return array Metadata with sources added.
```